test/unit/resources/aws_cloudwatch_alarm_test.rb

```diff
@@ -0,0 +1,167 @@
+require 'ostruct'
+require 'helper'
+require 'aws_cloudwatch_alarm'
+
+# MCWAB = MockCloudwatchAlarmBackend
+# Abbreviation not used outside this file
+
+#=============================================================================#
+#                            Constructor Tests
```

Comment:
In InSpec core we've historically we've used the `.must` ( e.g. `foo.must_equal` ) format instead of `assert`. For the sake of consistency, we might need to rewrite these. Thoughts?

Reply:
So, the inspec-aws project has been using dependency injection along with assertions, see #110 .  Work is currently being done to remove the DI stuff, which leaves it using the Minitest-class+assertions approach.  

My anticipation is that in a later issue, we'll cut over to using expectations-style (`must`), as used by inspec-core.  We could decide that all new resources (including this one) use expectations.  

I'm happy to make the change, just need a sense of when to make that decision.

Reply:
Is there any reason why we should not use `assert`? We are not proposing the syntax for InSpec users, because it is only understood well by programmers and the should syntax is more readable. I do not see an issue why we should not use this as part of our unit tests. I also think that we do not need to standardize on that, since both ways are useful depending on what you are trying to implement.

Reply:
@chris-rock I see an issue only in terms of consistency. I'd rather all our unit tests look-and-feel the same, and knowing this is going to be merged into core InSpec, I think we need to think carefully about maintaining a consistent style.